cms/templates/js/show-correctness-editor.underscore

```diff
@@ -0,0 +1,36 @@
+<form>
+<h3 class="modal-section-title" id="show_correctness_label"><%- gettext('Correctness Visibility') %></h3>
+<div class="modal-section-content show-correctness">
+    <radiogroup class="list-fields" aria-labelledby="show_correctness_label">
+        <label class="label">
+            <input class="input input-radio" name="show-correctness" type="radio" value="always" aria-describedby="always_show_correctness_description">
+            <%- gettext('Always show correctness') %>
+        </label>
+        <p class='field-message' id='always_show_correctness_description'>
+            <%- gettext('When learners submit an answer to a problem, they can immediately see whether the answer is correct or incorrect, and the score received.') %>
+        </p>
+        <label class="label">
+            <input class="input input-radio" name="show-correctness" type="radio" value="never" aria-describedby="never_show_correctness_description">
+            <%- gettext('Never show correctness') %>
+        </label>
+        <p class='field-message' id='never_show_correctness_description'>
+            <%- gettext('Learners are never allowed to see whether their answers to problems are correct or incorrect, or the score received.') %>
+        </p>
+        <label class="label">
+            <input class="input input-radio" name="show-correctness" type="radio" value="past_due" aria-describedby="show_correctness_past_due_description">
+            <%- gettext('Show correctness when subsection is past due') %>
+        </label>
+        <p class='field-message' id='show_correctness_past_due_description'>
+            <% if (self_paced) { %>
+                <%- gettext('Learners are not allowed to see whether their answers to problems were correct or incorrect, or the score received, until after the Course End Date has passed.') %>
+                <br />
```

Comment:
I would suggest just allowing the second sentence to flow immediately after the first one (instead of having a line break).